index.d.ts

```diff
@@ -4,24 +4,40 @@ import { AuthorizationParameters, TokenSet, UserinfoResponse } from 'openid-clie
 import { Request, RequestHandler } from 'express';
 
 interface ConfigParams {
+    appSessionCookie?: SessionCookieConfigParams;
+    appSessionDuration?: number;
+    appSessionName?: string;
+    appSessionSecret: boolean | string | string[];
     auth0Logout?: boolean;
     authorizationParams?: AuthorizationParameters
     baseURL?: string;
     clientID?: string;
     clientSecret?: string;
     clockTolerance?: number;
     errorOnRequiredAuth?: boolean;
-    getUser?: (tokenSet: TokenSet) => undefined | UserinfoResponse;
```

Comment:
could this be considered a breaking change? 🤔

Reply:
Yes, a big one! #42